common/scripted_triggers/wc_artifact_spawning_scripted_triggers.txt

```diff
@@ -9,5 +9,13 @@ THIS_is_generic_artifact_that_needs_dynamic_picture_trigger = {
 }
 
 THIS_is_generic_artifact_that_needs_function_words_trigger = {
-	has_artifact_flag = ranged_weapon
+	# OR = {
+		has_artifact_flag = ranged_weapon
+	# }
+}
+
+THIS_is_multiple_generic_artifacts_trigger = {
+	# OR = {
+		has_artifact_flag = bracelet
```

Comment:
By analogy with wonders, we can use `is_plural` flag.

Reply:
And if we use `set_flag`, `has_flag` these custom localization will be universal.

Reply:
done